### DISCUSSION TITLE
这个圣遗物怎么样

### DISCUSSION THREAD

**Discussion by swqs123** [share121/inter-knot#3337] [其他]
![Uploading Image_1723512543138.jpg…]()

**Comment by Nie7bai** [comment]
图片没有上传上来呢，要等Uploading没了才算上传完毕<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >
![tp2](https://github.com/user-attachments/assets/6596c08e-1949-44b7-b46b-c4cff3d21007)

**Comment by chihung3927** [comment]
![1724083591784](https://github.com/user-attachments/assets/fe2ae92d-c600-4dff-9162-23febb5e1a37)

**Comment by Glthubofficial** [comment]
圣遗物？旅行者，绝区零好玩吗
